### DISCUSSION TITLE
I can't download Blackhole 16 channel for Mac

### DISCUSSION THREAD

**Discussion by 302just-not-COOL** [ExistentialAudio/BlackHole#750] [Q&A]
When I click on "download installer" I get a screen that says " This site can't provide a secure connection". I've tried Chrome and Safari. I've updated the operating system. I've been with Apple support twice. They said it's not the computer. HELP!!!

**Comment by devinroth** [comment]
Someone emailed me with a similar issue. Turns out the issue was a mesh network blocking the website. Here’s what they said.

“Figured it out!!!  It was my Mesh wifi in the house that was blocking.  If anyone has the same problem, I installed the iOS app for the EERO mesh and turned off the blocking. “

Devin

**Comment by 302just-not-COOL** [comment]
Thanks Devin!
How do I know if I have a mesh network?
Thanks!
Doug Munro
https://dougmunro.com/
https://en.wikipedia.org/wiki/Doug_Munro_(musician)
https://www.youtube.com/channel/UCEYMtwRaMgghm18mqfaG5RA
https://www.facebook.com/dmunro302
https://twitter.com/dmunro302
https://www.instagram.com/danthonymunro/

**Reply by morrisschmorris** [reply]
i also am having this issue, whats the fix?

**Comment by 302just-not-COOL** [comment]
Check what Devin Ross said. I haven’t tried it. Cheers!

**Comment by devinroth** [comment]
All I can say is you have some sort of firewall, or something that is blocking your traffic. I will contact my webhost to see if there is some sort of security update that needs to happen to prevent this block.

Devin

**Reply by StrangeThistle** [reply]
ExistentialAudio seems to be flagged as a phishing/deception site in the EERO app, if that helps. Access to the firewall features is under the Discover tab. Turning off Advanced Security will let you through. I'm guessing the way the donation page is set up may seem a little sketchy.

**Reply by devinroth** [reply]
According to virustotal the only one flagging existential.audio is CRDF. EERO must be using that list. Working on removing the site from the list.

**Reply by devinroth** [reply]
CRDF has removed existential.audio from their list. Everything should be clear now.

**Comment by 302just-not-COOL** [comment]
Thank you Devin!!!
Doug Munro
https://dougmunro.com/
https://en.wikipedia.org/wiki/Doug_Munro_(musician)
https://www.youtube.com/channel/UCEYMtwRaMgghm18mqfaG5RA
https://www.facebook.com/dmunro302
https://twitter.com/dmunro302
https://www.instagram.com/danthonymunro/
